[Bug]: CNV is failing to be deployed on 5.0 OCP version.

### Description
We have automation issue to fix CNV 5.0 deployment.

### Steps to Reproduce
1. Install 5.0 With CNV deployment enabled.
2.
3.

### Actual Behavior
```
ocs_ci/deployment/deployment.py:1012: in deploy_cluster
    self.do_deploy_cnv()
ocs_ci/deployment/deployment.py:845: in do_deploy_cnv
    CNVInstaller().deploy_cnv(check_cnv_deployed, check_cnv_ready)
ocs_ci/deployment/cnv.py:722: in deploy_cnv
    self.create_cnv_catalog_source()
ocs_ci/deployment/cnv.py:84: in create_cnv_catalog_source
    cnv_catalog_source.wait_for_state("READY")
ocs_ci/utility/retry.py:99: in f_retry
    return f(*args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <ocs_ci.ocs.resources.catalog_source.CatalogSource object at 0x7fc311615e50>
state = 'READY', timeout = 480, sleep = 5

    @retry(ResourceWrongStatusException, tries=4, delay=5, backoff=1)
    def wait_for_state(self, state, timeout=480, sleep=5):
        """
        Wait till state of catalog source resource is the same as required one
        passed in the state parameter.
    
        Args:
            state (str): Desired state of catalog source object
            timeout (int): Timeout in seconds to wait for desired state
            sleep (int): Time in seconds to sleep between attempts
    
        Raises:
            ResourceWrongStatusException: In case the catalog source is not in
                expected state.
    
        """
        self.check_name_is_specified()
        sampler = TimeoutSampler(timeout, sleep, self.check_state, state=state)
        if not sampler.wait_for_func_status(True):
>           raise ResourceWrongStatusException(
                f"Catalog source: {self.resource_name} is not in expected "
                f"state: {state}"
            )
E           ocs_ci.ocs.exceptions.ResourceWrongStatusException: Resource Catalog source: cnv-nightly-catalog-source is not in expected state: READY

ocs_ci/ocs/resources/catalog_source.py:124: ResourceWrongStatusException
```
### Expected Behavior
Have CNV deployed




### Additional Context
 https://url.corp.redhat.com/9913eaa

--- a/ocs_ci/deployment/cnv.py
+++ b/ocs_ci/deployment/cnv.py
@@ -64,7 +64,7 @@ class CNVInstaller(object):
         logger.info("Adding CatalogSource for CNV")
         cnv_catalog_source_data = templating.load_yaml(CNV_CATALOG_SOURCE_YAML)
         cnv_catalog_source_name = cnv_catalog_source_data.get("metadata").get("name")
-        cnv_image_tag = config.DEPLOYMENT.get("ocs_csv_channel")[-4:]
+        cnv_image_tag = config.DEPLOYMENT.get("ocs_csv_channel").split("-")[-1]
         cnv_catalog_source_data["spec"][
             "image"
         ] = f"{constants.CNV_QUAY_NIGHTLY_IMAGE}:{cnv_image_tag}"
@@ -134,7 +134,9 @@ class CNVInstaller(object):
             ] = constants.OPERATOR_CATALOG_SOURCE_NAME
             cnv_sub_channel = "stable"
         else:
-            cnv_channel_version = config.DEPLOYMENT.get("ocs_csv_channel")[-4:]
+            cnv_channel_version = config.DEPLOYMENT.get("ocs_csv_channel").split("-")[
+                -1
+            ]
             cnv_sub_channel = f"nightly-{cnv_channel_version}"
 
         cnv_subscription_yaml_data["spec"]["channel"] = f"{cnv_sub_channel}"

